core/common/src/main/java/alluxio/master/PollingMasterInquireClient.java

```diff
@@ -46,9 +45,8 @@
 import javax.annotation.Nullable;
 
 /**
- * PollingMasterInquireClient finds the address of the primary master by polling a list of master
- * addresses to see if their RPC servers are serving. This works because only primary masters serve
- * RPCs.
+ * PollingMasterInquireClient finds the address of the primary master by
+ * polling a list of master addresses to see if they respond as the leader.
  */
 public class PollingMasterInquireClient implements MasterInquireClient {
```

Comment:
@jenoudet I didn't rename to **RaftPollingMasterInquireClient** because JournalMasterClientService::GetNodeState endpoint works for both UFS and Raft.